skyve-war/src/main/webapp/external/selfRegister.xhtml

```diff
@@ -75,6 +72,10 @@
 									childStyle="padding: 0px;"
 									childStyleClass="ui-g-12" />
 									
+								<!-- Password strength estimator -->
+								<script type="text/javascript" src="zxcvbn/zxcvbn-4.4.2-min.js"></script>
+								<link type="text/css" rel="stylesheet" href="zxcvbn/strength-bar-min.css"/>
+									
```

Comment:
Why are these imports with the view markup? I would put these with the other scripts at the bottom of the page? ~ Line 120

Reply:
No reason - just moved :)